### PR TITLE
Fixing template prefix when BASE_ENV is '/'

### PR DESCRIPTION
**Pull request by macartur**

### CODE REVIEW

**Comment by beraldoleal** [comment]
Duplicated of #402 ?

**Comment by erickvermot** [comment]
@macartur is right, #402 still outputs double slashes when BASE_ENV is '/'. I'll close #402, this is a better solution.

**Review by diraol — APPROVED** [review]

File: etc/kytos/kytos.conf.template [hunk 1]
```diff
@@ -21,15 +21,15 @@ daemon = False
...
+logging = {{ prefix }}etc/kytos/logging.ini
 
 listen = 0.0.0.0
 
 port = 6633
 
 # Where should the controller look for network apps ?
 # This directory has both core napps and user installed napps.
-napps = {{ prefix }}/var/lib/kytos/napps
+napps = {{ prefix }}var/lib/kytos/napps
```
Comment (beraldoleal) [inline etc/kytos/kytos.conf.template]: When we read things like this looks odd. I prefer `{{ prefix }}/var/lib/kytos/napps`. In order to do this, we have to make sure that prefix is something without a ending slash or something empty.

So if prefix == / lets make prefix == "". And additionally read my next comment.

File: setup.py [hunk 2]
```diff
@@ -128,6 +128,9 @@ def generate_file_from_template(cls, templates,
         """
         from jinja2 import Template
 
+        if not kwargs.get('prefix').endswith('/'):
+            kwargs['prefix'] += '/'
+
```
Comment (beraldoleal) [inline setup.py]: Based on the previous comment, lets do the oposite. If prefix ends with '/' lets remove the last '/'.

**Review by cemsbr — APPROVED** [review]

**Review by beraldoleal — CHANGES_REQUESTED** [review]

**Comment by beraldoleal** [comment]
@macartur ?

**Comment by macartur** [comment]
@beraldoleal  Fixed.

**Pull request merged**